scripts/before_install.sh

```diff
@@ -5,6 +5,7 @@ if (go run scripts/mingoversion.go 1.12 &>/dev/null); then
   go install -v golang.org/x/lint/golint
   go install -v golang.org/x/tools/cmd/goimports
   go install -v github.com/elastic/go-licenser
+  go get -v github.com/jcchavezs/porto
```

Comment:
```suggestion
  go install -v github.com/jcchavezs/porto
```

Reply:
Any reason not to make that change? I would prefer to have consistency

Reply:
I think I did not commit it by mistake.